odoo/addons/base/models/ir_http.py

```diff
@@ -114,6 +114,19 @@ class IrHttp(models.AbstractModel):
     _name = 'ir.http'
     _description = "HTTP Routing"
 
+    @classmethod
+    def _slug(cls, value: models.BaseModel | tuple[int, str]) -> str:
+        if isinstance(value, tuple):
+            return str(value[0])
+        return str(value.id)
```

Comment:
Mmh I guess my previous comment is wrong since you are adding this in base? Should it be squashed? :shrug:

Reply:
The slug that is there corresponds to the parsing of URLs done when there is only base.

Reply:
I understood that, see previous comment: should it be squashed? Otherwise, first commit is wrong.

Reply:
I've separated the commits for easier review. For me the merge could be squashed